README.md

```diff
@@ -75,15 +75,36 @@ The best way to develop the package is to open the demo app Xcode project
 This is one of the tradeoffs of using SwiftPackageManager, in standalone packages we cannot have demo apps, neither configure Xcode to run SwiftLint. So we added this to the DemoApp.
 
 ## Releasing
+In order to release a new version of the package (and SATSCore iOS app)
 
-In order to release a new version of the package (and SATSCore iOS app) we just need to add a tag in the [Releases section](https://github.com/healthfitnessnordic/SATSCore-iOS/releases).
+1. Create a PR for your improvements
+2. Merge your PR to main
+3. Get latest released version from the [releases section]
 
-Then make sure the `Tag` and `Title` of the release is in the form `0.0.8` or similar as this number is used by SPM to define the version.
+4. Get changelog from last version which in this case is `1.3.0`
```

Comment:
```suggestion
4. Get the changelog from latest version which in this case is `1.3.0`
```

Reply:
```suggestion
4. Get the changelog from the latest version which in this case is `1.3.0`
```

Reply:
well google translate  agree with ´the latest´